tracer/src/Datadog.Trace.Tools.Runner/RunHelper.cs

```diff
@@ -51,6 +51,7 @@ public static int Execute(ApplicationContext applicationContext, CommandContext
             // If the agentless feature flag is enabled, we check for ApiKey
             // If the agentless feature flag is disabled, we check if we have connection to the agent before running the process.
             var createTestSession = false;
+            var testSkippingDisabled = false;
```

Comment:
nit: could we invert this variable, `testSkippingEnabled`? I think it would make it easier to follow, especially as the other settings are all "enabled" settings

Reply:
done in [f49bda7](https://github.com/DataDog/dd-trace-dotnet/pull/3524/commits/f49bda7ec33fe05db6300702baba58ee4641ff2e)